### ISSUE TITLE
Using multivariate statistical techniques to investigate the role of brain volume and neurite density on psychopathology

### ISSUE THREAD

**Issue by kharloews** [ABCD-ReproNim/projects#24]
## Research question(s)

How do brain volume (i.e., cortical grey matter volume, white matter tract volume, subcortical volume) interact with neural measures of cell density (i.e. RSI) to explain adolescent psychopathology? 

Using multivariate statistical analysis techniques (e.g. partial least squares, generalized canonical correlations), can we uncover different profiles in brain volume/RSI measures and psychopathology?

## Description

The purpose of this project by Claire Campbell and Carlos Cardenas-Iniguez is to investigate how measures of brain volume and several Restricted Spectrum Imaging (RSI) in wave 1 relate to measures of psychopathology across a number of categories. Importantly, we would like to use multivariate techniques such as PLS and CCA to see if we can uncover multiple latent variables that describe the relationship between RSI and volume measures and syndrome scores derived from the Child Behavioral Checklist. 

We are planning to use RSI and volume estimates from the tabulated data across three sets of ROIs that cover cortical grey matter, several white matter tracts, and subcortical structures and use PLS/GCCA to relate these to continuous psychopathology scores. We hope to uncover latent variables that show relationships between measures in these brain regions and various psychopathology scores.

If time allows, we hope to see how relationships in the extracted latent variables relate to several sociodemographic and environmental variables. Also (if time allows), we hope to use SEM to estimate a bifactor model of psychopathology for the CBCL scores to investigate the same research questions. 

We plan to use the ABCD Reproducible Matched Samples (ARMS) groups so as to run our models across two samples. 
 
### Measures of interest, from tabulated data (Release 3.0):
Cortical volume measures using Desikan-Killiany parcellation
Fiber tract volume measures using AtlasTrack labels
Subcortical volume measures (ASEG)
Hindered normalized isotropic (N0; F0 by T) measures of RSI model fit for ASEG, APARC, and AtlasTrack ROIs
Hindered normalized anisotropic (ND; FD by T) measures of RSI model fit for ASEG, APARC, and AtlasTrack ROIs
Child Behavioral Checklist (CBCL) raw scores and syndrome scores

## Tools and algorithms to be used

For data cleaning: R
Analysis: Partial Least Squares (MATLAB using code referenced in McIntosh & Lobaugh, 2004)  Generalized Canonical Correlation Analysis (using custom MATLAB code or RGCCA package in R)
To visualize brain scores from the model: ggseg package in R for ASEG and APROC volumes, and DTI studio to visualize scores for AtlasTrack-labeled WM tracts
[Possible] Mplus or lavaan (in R) to estimate subject-level factor scores after estimating a bifactor model using CBCL items

## Skills we could use help with (optional)

While our group has experience running PLS and CCAs in MATLAB, any suggestions on useful tools/packages to run these in R or python (preferably) are very much appreciated--especially for permutations/bootstrapping to get reliability metrics on estimates. 
Any other suggestions on tools to visualize scalar values from our models on to ASEG and APARC volumes would be very useful!
We will be using the ARMS matched groups to run our models twice, but would also welcome any other suggestions for evaluating our models (such as cross-validation, etc). 

## Suggested keywords/tags

Psychopathology
PLS
CCA
CBCL
Cortical Volume

**Comment by satra** [comment]
For several of the above analysis, you may want to look into scikit-learn and statsmodels in python. If you intend to go into the more Bayesian route, you can look into tensorflow-probability.

**Comment by llevitis** [comment]
This project sounds awesome, and I'd be happy to help out or serve as a TA point person :) 
Following up on Satra's suggestion for Python libraries, you can certainly use Nilearn if you'd like to visualize ROI-level scores using Python.
